samples/40.timex-resolution/src/main/java/com/microsoft/bot/sample/timex/resolution/Parsing.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.bot.sample.timex.resolution;
+
+import com.microsoft.recognizers.datatypes.timex.expression.Constants.TimexTypes;
+import com.microsoft.recognizers.datatypes.timex.expression.TimexProperty;
+
+public class Parsing
+{
+    private static void Describe(TimexProperty t)
```

Comment:
```suggestion
    private static void describe(TimexProperty t)
```

Methods in Java are lowerCamelCase